postgres/src/test/java/org/jdbi/v3/postgres/TestJavaTime.java

```diff
@@ -45,6 +49,15 @@ public void setUp() {
         });
     }
 
+    private TemporalUnitOffset getAllowableOffset() {
+        // PostgreSQL seems to not have as much precision on Windows as it does on Linux.
+        if (SystemUtils.IS_OS_WINDOWS) {
+            return within(0, ChronoUnit.MICROS);
```

Comment:
Can we just use the more lenient comparison window always, and get rid of the OS detection?  We are not really intending to test the system clock here at all, and OS-specific behavior should be avoided whenever possible.

Reply:
Yup I'll just remove the condition.